### PR TITLE
slack: add audit.context.session_id

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Adds handling of the audit context `session_id` field used to uniquely identify sessions.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Clsoes #6189

### CODE REVIEW

File: packages/slack/data_stream/audit/fields/fields.yml
Comment (christophercutajar) [inline packages/slack/data_stream/audit/fields/fields.yml:1]: @efd6 I was looking at what changes are required for the issue I raised last Friday and also noticed this typo. It is just a cosmetic thing but would be nice to have it corrected. Change from `Cloudflare` to `Slack`

**Comment by efd6** [comment]
Yeah, conventionally ID *are* keywords. Is there any indication that session_ids are assigned contiguously, if that were true, then keeping as long might provide some benefit, otherwise I'll change to a keyword. Let me know,

**Comment by SpencerLN** [comment]
I've looked through our logs, and I don't see any obvious pattern that would indicate that session_ids are contiguous or not otherwise random.

**Review by ebeahan — APPROVED** [review]
LGTM

**Pull request merged**